src/http/constants.h

```diff
@@ -0,0 +1,35 @@
+#ifndef STATUS_H
+#define STATUS_H
+
+#include <iostream>
+#include <string>
+
+/**
+ * Non-exhaustive constants file for different HTTP status
+ * These status are added on an at need basis. 
+ * Currently the only supported statuses are 200 and 404
+ * 
+ * Heavily inspired by
+ * https://github.com/tomchristie/django-rest-framework/blob/master/rest_framework/status.py
+ */
+namespace status {
+	const int HTTP_200_OK = 200;
+	const int HTTP_404_NOT_FOUND = 404;
+
+	inline std::string getReasonPhrase(int fromStatus) {
+		switch(fromStatus) {
+			case HTTP_200_OK: return "OK";
+			case HTTP_404_NOT_FOUND: return "NOT FOUND";
+			default: 
+				std::cerr << "invalid status code" << std::endl;
+				exit(-1); 
```

Comment:
Technically we should have more defined error codes, but I'll change them to 1 for consistency as a general error and we can specialize error codes later if we need it